### DISCUSSION TITLE
General Question(s)

### DISCUSSION THREAD

**Discussion by gijoe2cool** [hxr404/Discord-ExpireBot#6] [General]
Just curious as to how this is going, I'd love this feature since as I'm sure you know no bots really do this, and I've been looking. So just curious as to how it's progressing. 

Also, idk how/if I could help, but I'd be willing to help out if I could, and if it matter I'm a web designer and have a server/host we could use if needed/if I'm able to. Just tossing that out there if it helps in any way.

**Comment by hxr404** [comment]
Hi, I really appreciate your help, as I'm really busy at the moment and theres a lot to do
heres a quick oveview:
The Bot should work propery now, but hasn't got many features. It can do:
- set roles to expire of course and remove them again
- admin can set a role which can configure the bot
- native Linux support (easy config)
- native Windows support (easy config)

what I plan to do:
- ~~windows support (easy config)~~ Edit: Already did that
- ~~display config command~~ Edit: Already did that but needs more testing
- ~~multi guild support~~ Edit: started working on it.
- sharding

so to make the bot public, I first have to implement multi guild support
and after this, hosting would also be a problem, idk if a raspi would be enough, so I thought about doing a community-hosted bot, everyone can apply for hosting a shard, but idk yet. I'll open a new issue for that thing

If you want to contribute to the code you can create a pull request. the four things mentioned above are the most important thing, but ofc you can also suggest other features
